_episodes/releases/2022-10-05-rust-1.62-1.63-1.64.md

```diff
@@ -0,0 +1,123 @@
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+#### [@00:50] - [Rust 1.62](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html)
+
+ - [@00:58] - [`cargo add`](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html#cargo-add)
+   - [Maintaining sorting in TOML files](https://github.com/rust-lang/cargo/blob/13ae438cf079da58272edc71f4d4968043dbd27b/src/cargo/ops/cargo_add/mod.rs#L97)
+   - [`toml_edit`](https://crates.io/crates/toml_edit)
+   - [`cargo-edit`](https://github.com/killercup/cargo-edit)
+ - [@03:37] - [`#[default]` enum variants](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html#default-enum-variants)
+   - [Generated bounds part of RFC](https://rust-lang.github.io/rfcs/3107-derive-default-enum.html#generated-bounds)
+   - [Macro helper attributes](https://doc.rust-lang.org/stable/reference/procedural-macros.html#derive-macro-helper-attributes)
+   - [Extra bounds on derive](https://github.com/rust-lang/rust/issues/26925)
+ - [@07:36] - [Thinner, faster mutexes on Linux](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html#thinner-faster-mutexes-on-linux)
+   - [Tracking issue](https://github.com/rust-lang/rust/issues/93740)
+   - [Short thread on the change from Mara](https://twitter.com/m_ou_se/status/1526211117651050497)
+   - [More details from Mara on pthread mutexes](https://twitter.com/m_ou_se/status/1505179030336819209)
+ - [@13:21] - [Bare-metal x86_64 target](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html#bare-metal-x86_64-target)
+   - [Target triples](https://wiki.osdev.org/Target_Triplet)
+   - [Tier 2 target policy](https://doc.rust-lang.org/rustc/target-tier-policy.html#tier-2-target-policy)
+   - [Tier 2 targets](https://doc.rust-lang.org/rustc/platform-support.html#tier-2)
+   - [x86_64-unknown-none](https://doc.rust-lang.org/beta/rustc/platform-support/x86_64-unknown-none.html)
+   - [Custom target triples](https://rustc-dev-guide.rust-lang.org/building/new-target.html)
+ - [@22:20] - [Stabilized APIs](https://blog.rust-lang.org/2022/06/30/Rust-1.62.0.html#stabilized-apis)
+   - [`f64::total_cmp`](https://doc.rust-lang.org/stable/std/primitive.f64.html#method.total_cmp)
+     - [Implementing PR](https://github.com/rust-lang/rust/pull/72568)
+   - [`Stdin::lines`](https://doc.rust-lang.org/stable/std/io/struct.Stdin.html#method.lines)
+   - [`FusedIterator`](https://doc.rust-lang.org/stable/std/iter/trait.FusedIterator.html)
+ - [@29:22] - Changelog deep-dive
+   - [`cargo -F` for `--features`](https://github.com/rust-lang/cargo/pull/10576)
+   - [`unaligned_references` lint now warns by default](https://github.com/rust-lang/rust/pull/95372/)
+     - [`addr_of!`](https://doc.rust-lang.org/core/ptr/macro.addr_of.html)
+
+#### [@31:09] - [Rust 1.62.1](https://blog.rust-lang.org/2022/07/19/Rust-1.62.1.html)
+
+Not much to talk about. We also didn't talk about
```

Comment:
Incomplete.

Reply:
Ah, no, it was intended as a lead in to the list below. Imagine a : there :p